core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -101,6 +101,11 @@
 
     private boolean hideInaccessibleColumns;
     private boolean forceSpillingJoin;
+    /**
+     * This value is purposefully identical to that of IncrementalLoadFactorHashArraySizeSupplier#THRESHOLD_50,
```

Comment:
Maybe you just reuse `io.trino.operator.IncrementalLoadFactorHashArraySizeSupplier#THRESHOLD_50`